dev.sh

```diff
@@ -4,6 +4,13 @@ VENV=./venv
 
 python -m virtualenv $VENV --always-copy
 . $VENV/bin/activate
+pip_version=$(pip -V | cut -d' ' -f 2 | cut -d'.' -f 1)
+if [ "$pip_version" -lt 6 ];
```

Comment:
Did you check if `6.0` is actually enough?
Maybe add a comment that mentions the reason (because Ubuntu 12.04 and 14.04 ship something older)